server/services/user-validation.ts

```diff
@@ -1,7 +1,14 @@
 import { getParticipantByID } from './participants';
 import logger from '../logger';
 
-const validateParticipant = async (participantId, actionType) => {
+interface User {
+  email?: string;
+  // eslint-disable-next-line camelcase
+  user_id: string;
+  sub: string;
```

Comment:
In this situation - user_id is referencing the `user.body.userInfo.id` not `user.id`, therefore it's a string not a number?

Reply:
This isn't an issue with your pr, I'm just noticing the inconsistencies between IDs being numbers vs strings (Which makes sense when we're referencing the body of the data and not the PK)

Reply:
Yeah, IDs are... messy. Not a fan of how they're handled right now. Potential refactor opportunity, but comes with a potential need for DB work.

Reply:
yeah a large refactor indeed.